MailManager/src/main/java/inc/softserve/EmailServiceConfiguration.java

```diff
@@ -17,6 +14,14 @@
     @JsonProperty
     String sendEmailPath;
 
+    @NotEmpty
+    @JsonProperty
+    String ESPConnectorAdmin;

```

Comment:
Java conventions do not recommend to start properties' names from capitals. Please consider to change it.